applications/debug/unit_tests/dialogs/dialogs_file_browser_options.c

```diff
@@ -0,0 +1,25 @@
+#include <dialogs/dialogs.h>
+
+#include "../minunit.h"
+
+MU_TEST(test_dialog_file_browser_set_basic_options_should_init_all_fields) {
```

Comment:
This test is incorrect: DialogsFileBrowserOptions structure is not packed and filling it with `n` is not going to work

Reply:
Hmm, I forgot about alignment. In this case, I am not sure what the right way to test this (allowing automatic updates in case of futue changes) would be.

Reply:
@skotopes, here is the alternative approach to the test. It may not handle some specific corner-cases but it is better than nothing.

It adds an assertion on structure size (guess I calculated it correctly) which would get triggered on most changes to `DialogsFileBrowserOptions` (except for ones which fit into alignment) forcing the developer to check the test and thus update initialization logic.